src/service/stamp/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Bee } from '@ethersphere/bee-js'
+import { exit } from 'process'
+import { CommandLog } from '../../command/root-command/command-log'
+
+export async function pickStamp(bee: Bee, console: CommandLog): Promise<string> {
```

Comment:
please make a jsdoc what it returns exactly

Reply:
```
/**
 * Displays an interactive stamp picker to select a Stamp ID.
 *
 * A typical use case is to prompt the user for a stamp, when
 * the command requires one, but was not specified in `argv`.
 *
 * Makes a request via `bee` to fetch possible stamps.
 *
 * @returns {Promise<string>} Hex representation of the Stamp ID.
 */
```